backend/pretix/__init__.py

```diff
@@ -384,3 +386,18 @@ def get_user_tickets(conference: Conference, email: str):
     )
 
     return response.json()
+
+
+def update_ticket(conference: Conference, attendee_ticket: UpdateAttendeeTicketInput):
+    print("Updating Ticket")
```

Comment:
make sure you remove all those prints once done

to debug you can also use `breakpoint()` that might be easier than having to look at the console